docs/includes/protections.md

```diff
@@ -139,6 +141,27 @@ def protections(self):
     This Protection applies only at pair-level, and will never lock all pairs globally.
     This Protection does not consider `lookback_period` as it only looks at the latest trade.
 
+#### Profit Limit
+
+`ProfitLimit` uses all trades for a pair within `lookback_period` in minutes (or in candles when using `lookback_period_candles`) to determine the overall profit ratio.
+If that ratio is greater than `profit_limit`, all pairs will be locked for `stop_duration` in minutes (or in candles when using `stop_duration_candles`).
```

Comment:
I'm not entirely sure if the current stop-duration concept (the way we use it for other protections) works for this limitation.

A common usecase I've heard is the possibility to "stop trading for today" once a certain profit has been reached.
I'd however not know how to configure this at this point - if i use "stop_duration" 180 (3h - as in the example) - it'd work fine if i reached the profit at 9pm - but it'll fail if i have the required 1% profit already at 3am - as the bot would continue trading after 6am (3 hours after the lock).